tests/native_tests/language.hy

```diff
@@ -340,19 +340,29 @@
 
 (defn test-cond []
   (cond
-   [(= 1 2) (assert (is True False))]
-   [(is None None) (setv x True) (assert x)])
+    (= 1 2) (assert (is True False))
+    (is None None) (do (setv x True) (assert x)))
   (assert (= (cond) None))
```

Comment:
also can we get this `=` changed to `is` while we're here?

Reply:
You mean changing `(= 1 2)` to `(is 1 2)`? Aren't the semantics of `is` with numbers a bit dicey?

Reply:
Oh no the `(= (cond) None)` one

Reply:
Oh, right. My eyeballs just skipped over that line because I hadn't edited it. Done.